.github/workflows/quality.yml

```diff
@@ -23,10 +23,13 @@ jobs:
           key: ${{ runner.os }}-modules-${{ hashFiles('**/pnpm-lock.yaml') }}
 
       - name: Install deps
-        run: pnpm i
+        run: pnpm i --r
```

Comment:
Should 100% regress this, `pnpm i` is by default workspace wide in the root, that wasn't the issue.